packages/fast-usdc-contract/src/fast-usdc.contract.ts

```diff
@@ -318,12 +322,49 @@ export const contract = async (
     remediateUndetectedBatches(minUusdc: bigint): void {
       return settlerKit.creator.remediateMintedEarly(minUusdc);
     },
-    /** @type {typeof chainHub.updateChain} */
-    updateChain(chainName, chainInfo) {
+    /**
+     * Transactions that reached FAILED_FORWARD status before we had a retrier
+     * (and stored failed forwards) were terminal and required manual payment.
+     * OpCo made those transfers from its own funds. This method is to be called
+     * in a CoreEval to reimburse those payments.
+     *
+     * @param agoricRecipient - The Bech32 of the recipient (must be Agoric).
+     * @param amount - The amount to send, in the USDC brand.
+     */
+    async sendFromSettlementAccount(
+      agoricRecipient: Bech32Address,
+      amount: AmountArg,
+    ): Promise<{ before: DenomAmount[]; after: DenomAmount[] }> {
+      trace(
+        `Sending ${quote(amount)} to ${agoricRecipient} from settlementAccount`,
+      );
+      const recipient = chainHub.resolveAccountId(agoricRecipient);
+      const before = await vowTools.when(E(settlementAccount).getBalances());
+
+      return vowTools.when(
+        E(settlementAccount).send(recipient, amount),
```

Comment:
I think we should have explicit justification for use of `vowTools.when(...)` in production contracts:

```suggestion
      // when: .send() on a local account such as the settlement account is prompt
      return vowTools.when(
        E(settlementAccount).send(recipient, amount),
```